manifests/base/kustomization.yaml

```diff
@@ -4,7 +4,6 @@ kind: Kustomization
 resources:
   - jupyter-minimal-notebook-imagestream.yaml
   - jupyter-datascience-notebook-imagestream.yaml
-  - jupyter-habana-notebook-imagestream.yaml
```

Comment:
this line should remain, no?

Reply:
Yes if we move the label to false we can do that...